common/pulp/common/constants.py

```diff
@@ -48,8 +48,12 @@
 PRIMARY_ID = '___/primary/___'
 
 # this is used by both platform and plugins to find the default CA path
-DEFAULT_CA_PATH = '/etc/pki/tls/certs/ca-bundle.crt'
-
+DEFAULT_CA_PATH_LIST = [
```

Comment:
It might be reasonable to leave this as `DEFAULT_CA_PATH`, and set its value at run-time with `get_from_default_paths`. Otherwise, it doesn't seem useful to define this as a constant if the only useful way to access it is by calling `get_from_default_paths` on its value. Part of the motivation for this is that this constant is imported by the RPM plugin, so this change will break that plugin, maybe more.

Maybe something like this is reasonable?
```python
DEFAULT_CA_PATH_LIST = [...]
# set DEFAULT_CA_PATH based on DEFAULT_CA_PATH_LIST for backward compatibility with plugins
for path in DEFAULT_CA_PATH_LIST:
    if os.path.exists(path):
        DEFAULT_CA_PATH = path
        break
else:
    DEFAULT_CA_PATH = None
```

I've specifically avoided using `get_from_default_paths` here; just about everything in pulp imports from constants, which would create circular import problems almost anywhere the function lived.